pmd-javascript/src/main/resources/category/ecmascript/errorprone.xml

```diff
@@ -6,9 +6,45 @@
          xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
 
     <description>
-
+Rules which flag certain language construct which can lead to abnormal behavior at runtime.
```

Comment:
Maybe a little harsher on this one as to what is "an abnormal behavior"

Rules to detect constructs that are either broken, extremely confusing or prone to runtime errors?